### ISSUE TITLE
Cannot open app UI after v2.1 release

### ISSUE THREAD

**Issue by Salat1K69** [Kataiser/tf2-rich-presence#206]
Been using this rich presence for some time now, worked pretty good (except the things at the bottom of this post), but after v2.1 the whole UI for the rich presence is broken. I simply cannot alt tab, or click on the app when its started.
![obrázok_2021-10-10_124123](https://user-images.githubusercontent.com/69127872/136692215-99e2e690-de40-4b41-9c84-5396f1e3c338.png)
![image](https://user-images.githubusercontent.com/69127872/136692274-d778a344-e33d-4c64-ba23-f801ac893aa8.png)

Other things i have noticed:

- Class is always "unselected" (Tried re-installing the RP app and TF2, didn´t help).
- Sometimes after disconnecting from a server, it will show the server (instead of being in the main menu) info untill i requeue or join another server.
- The kill counter didn´t work once, fixed it by just restarting TF2 (Seems to be fixed now).

Off-topic: Do you need a translator by any chance?

**Comment by Salat1K69** [comment]
The RP UI worked this time, not sure what was the problem. I did launch the RP before TF2 this time, and i used another shortcut. Maybe that did something ? (It now works every time, no matter if i launch the game or the RP first)

Can I attach the logs into this chat thingy/box ? 

Regarding the translations, I would like to translate the RP into the Czech and Slovak language. Is there like a .txt file with all the words used in the RP or anything similar?

The classes still seem to just be "unselected" (I tried messing with the settings a bit)

Also sorry for any dumb questions 😅
